safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -144,13 +144,10 @@ def run(self):
 
         displacement_rate = self.hardcoded_parameters['displacement_rate']
 
-        # Extract input layers
-        intensity = self.hazard
-        population = self.exposure
-
         # Extract data grids
-        hazard = intensity.get_data()   # Ground Shaking
-        exposure = population.get_data(scaling=True)  # Population Density
+        hazard = self.hazard.layer.get_data()   # Ground Shaking
```

Comment:
IMHO ```get_``` prefixes are superfluous (I appreciate it is not part of your patch, so just a general comment for future...)